test/integration/patients/patient/patient-flow.js

```diff
@@ -2983,6 +2990,45 @@ context('patient flow page', function() {
       .find('.table-list__item .fa-paperclip')
       .should('not.exist');
 
+    cy.sendWs({
+      category: 'CommentAdded',
+      author: getCurrentClinician().id,
+      resource: {
+        type: 'patient-actions',
+        id: testSocketAction.id,
+      },
+      payload: {
+        comment: {
```

Comment:
So I even think we should be using `getComment` here instead of `uuid` and then doing 
```
comment: {
  type: testComment.type,
  id: testComment.id,
}
```
we nearly have no instances of `type: 'patient-actions'` except for in this file for ws tests..  we probably want to change any instance of `type: '`

Reply:
I updated most of these `type: ''`'s in the file.

With the exception of `type: 'files'`. We don't currently have a `getFile()` to make that happen. Do we want one?